Dockerfile.windows

```diff
@@ -249,19 +286,21 @@ RUN `
   Remove-Item C:\binutils.zip; `
   Remove-Item C:\gitsetup.zip; `
   `
-  Write-Host INFO: Creating source directory...; `
-  New-Item -ItemType Directory -Path ${GOPATH}\src\github.com\docker\docker | Out-Null; `
+  # Ensure all directories exist that we will require below....
+  $srcDir = """$Env:GOPATH`\src\github.com\docker\docker\bundles"""; `
+  Write-Host INFO: Ensuring existence of directory $srcDir...; `
+  New-Item -ItemType Directory -Path $srcDir | Out-Null; `
```

Comment:
We should probably add `-Force` here (to get the equivalent of `mkdir -p`); we don't want this to fail if the `bundles` directory exists.

Reply:
Done